### DISCUSSION TITLE
Toolbar on Windows 10 taskbar

### DISCUSSION THREAD

**Discussion by TimH32** [valinet/ExplorerPatcher#4033] [General]
Just updated to Windows 11 24H2 and although EP for the most part is working I've lost the ability to have Windows 10 toolbars.

Does anyone know if there are any plans for toolbar support to return or is it gone for good?

Primary reason I've been using EP was to get back the quick launch toolbar that Microsoft, in their infinite wisdom, decided to completely remove all toolbars in Windows 11.  Grrrrr

**Comment by pyrates999** [comment]
Open up EP Properties application, go to taskbar, and set **taskbar style** to **Windows 10 (ExplorerPatcher)**, _not_ **Windows 10**.  We no longer offer more support on the stock Windows 10 taskbar.

please see this for more info: https://github.com/valinet/ExplorerPatcher/wiki/ExplorerPatcher's-taskbar-implementation

If you don't see this option, you need to update your version of EP that is available here: https://github.com/valinet/ExplorerPatcher/releases

**Reply by TimH32** [reply]
Thanks for replying.

I've got EP properties taskbar style set to Windows 10 (ExplorderPatcher)... when I right click the taskbar I don't see the "Toolbars" option any longer like I did before upgrading to 24H2.

So am I just SOL and the Toolbars option is gone for good?

**Comment by TimH32** [comment]
Hold up... apparently I just hadn't rebooted enough times after upgrading to 24H2.

Suddenly after about the 4th reboot since the upgrade to 24H2 I can now see the Toolbars option again and was able to add the quick launch toolbar back to my taskbar. Woo Hoo!

**Reply by pyrates999** [reply]
Glad it works now :)
